test/setup.ts

```diff
@@ -1,3 +1,4 @@
+require('dotenv').config();
```

Comment:
Tests already have a way to call dotenv in the require command (`node -r dotenv/config`). Remove this line.

Reply:
nice!